devTools/configUploader.js

```diff
@@ -12,15 +12,17 @@ const SMCloudStore = require('smcloudstore');
     const storagePath= env('STORAGE_PATH');
     const storageContainer= env('STORAGE_CONTAINER');
     const storageConnection = (env('STORAGE_CONNECTION') && JSON.parse(env('STORAGE_CONNECTION')))
-    const configType = env('REMOTE_CONFIG_TYPE') || 'json';
-    console.log("configType", configType)
+    const configType = env('REMOTE_CONFIG_TYPE') || 'local';
```

Comment:
Why `local` is now the default config type? It should be `json`.

Reply:
local means local json or js
json or js means remote json & js